consumers/tracing/consumers.go

```diff
@@ -15,31 +17,46 @@ const (
 	consumeAsyncOP    = "consume_async_op"
```

Comment:
This operation name is not a standard operation name supported by open telemetry. I'd suggest something like
```go
consumeAsyncOP    = "receive_async" // This is not specified in the open telemetry spec.
```

Reply:
@SammyOina also add the comments as it is not specified operation name